src/lib/plugins/system-info.js

```diff
@@ -0,0 +1,517 @@
+// This library contains several chunks of code from Sebastian Hildebrandt's
+// systeminformation library, MIT
+// https://github.com/sebhildebrandt/systeminformation
```

Comment:
I agree, I think we should consume this library as a dependency if at all possible